common/thrift_router.cpp

```diff
@@ -45,8 +58,10 @@ bool parseHost(const std::string& str, common::detail::Host* host) {
   } catch (...) {
     return false;
   }
-
-  host->az = (tokens.size() == 3 ? tokens[2] : "unknown");
+  auto group = (tokens.size() == 3 ? tokens[2] : "unknown");
```

Comment:
This is not introduced by your change. Does it make sense to replace "unknown" with ""?